client/state/ui/editor/actions.js

```diff
@@ -154,6 +156,11 @@ export function saveConfirmationSidebarPreference( siteId, isEnabled = true ) {
 	};
 }
 
+export const setEditorIframeLoaded = ( loaded = true ) => ( {
+	type: EDITOR_IFRAME_LOADED,
+	isIframeLoaded: loaded,
```

Comment:
nit: if you named the function parameter `isIframeLoaded` instead of `loaded`, you could use a property shorthand syntax 🙂

Reply:
Done in 42f4f01